cats/configure.py

```diff
@@ -76,7 +77,16 @@ def config_from_file(configpath="") -> Mapping[str, Any]:
             return yaml.safe_load(f)
         logging.info(f"Using provided config file: {configpath}\n")
     else:
-        # if no path provided, look for `config.yml` in current directory
+        # if no path provided, try to use config environment variable
+        cfile = os.getenv("CATS_CONFIG_FILE")
+        if cfile is not None:
+            try:
+                with open(cfile, "r") as f:
+                    return yaml.safe_load(f)
+                logging.info("Using config.yml found in CATS_CONFIG_FILE\n")
+            except FileNotFoundError:
+                logging.warning("CATS_CONFIG_FILE config file not found")
```

Comment:
Would benefit from a test being added to test_configure.py (which will stop codecov throwing a fit). I'll see if I can add something to this PR for that.

Reply:
I've created a PR against your main branch - I _think_ that will add a test to this PR if you merge it.